proc/process.c

```diff
@@ -395,26 +395,185 @@ static void process_tlsAssign(hal_tls_t *process_tls, hal_tls_t *tls, ptr_t tbss
 }
 
 
+static int process_isPtrValid(void *mapStart, size_t mapSize, void *ptrStart, size_t ptrSize)
+{
+	void *mapEnd = ((char *)mapStart) + mapSize;
+	void *ptrEnd = ((char *)ptrStart) + ptrSize;
+
+	/* clang-format off */
+	return ((ptrSize == 0) ||
+			((ptrEnd > mapStart) && (ptrEnd <= mapEnd) &&
+				(ptrStart >= mapStart) && (ptrStart < ptrEnd))) ?
+			1 : 0;
+	/* clang-format on */
+}
+
+
+/* Ensure kernel won't make a bad access on a malformed elf during load. */
+static int process_loadValidateElf32(void *iehdr, size_t size)
```

Comment:
I get what you were trying to do, but we don't load anything here
```suggestion
static int process_validateElf32(void *iehdr, size_t size)
```